php8.3/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM debian:buster
```

Comment:
should it not be `debian:bookworm` ?

Reply:
Hmm, yah might be a good idea to update the image. Works for now but buster is EOL on x86_64.